qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -506,6 +508,15 @@ def backends(
                         filters=lambda b: b.max_shots > 50000)
                     QiskitRuntimeService.backends(
                         filters=lambda x: ("rz" in x.basis_gates )
+            use_fractional_gates: Set True to allow for the backends to include
+                fractional gates in target. Currently this feature cannot be used
+                simulataneously with the dynamic circuits.
```

Comment:
We should also point out it doesn't work with PEA/PEC. So if a user plans to enable those features, they should transpile with a target that doesn't include fractional gates. 

```suggestion
            use_fractional_gates: Set True to allow for the backends to include
                fractional gates in target. Currently this feature cannot be used
                simulataneously with the dynamic circuits, PEC, or PEA.
```